projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+export interface IRunJobDialogProps {
+    /**
+     * Current Job path
+     */
+    jobPath: string;
+}
+
+
+export default class RunJobDialong extends Component<IRunJobDialogProps> {
```

Comment:
nit: it still says "Dialong" instead of "Dialog"

Reply:
I will fix it

Reply:
it is fixed here: https://github.com/vmware/versatile-data-kit/pull/1454/commits/76c97c63089ceb52056539a126ce21d58dd51228